limited_area/limited_area.py

```diff
@@ -137,30 +168,40 @@ def gen_region(self, *args, **kwargs):
                                            **kwargs)
 
 
-        # Subset the grid into a new region:
-        print('Subsetting mesh fields into the specified region mesh...')
-        regionFname = self.create_regional_fname(name, self.mesh)
-        regionalMesh = self.mesh.subset_fields(regionFname,
-                                          bdyMaskCell,
-                                          bdyMaskEdge,
-                                          bdyMaskVertex,
-                                          inside=self.INSIDE,
-                                          unmarked=self.UNMARKED,
-                                          format=self.cdf_format,
-                                          *args,
-                                          **kwargs)
-
-        print('Copying global attributes...')
-        self.mesh.copy_global_attributes(regionalMesh)
+        # Create subsets of all the meshes in self.meshes
+        print('Subsetting meshes...')
+        for mesh in self.meshes:
+            print("\nSubsetting:", mesh.fname)
```

Comment:
Should we change this print message to say `print("Subsetting:", mesh.fname, "using", mesh.fname)`.